src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -237,11 +247,23 @@ export class StudySummaryTab extends React.Component<
                     props.getData = () =>
                         this.store.getChartDownloadableData(chartMeta);
                 } else {
-                    props.promise = this.store.getClinicalDataBin(chartMeta);
-                    props.filters = this.store.getClinicalDataFiltersByUniqueKey(
-                        chartMeta.uniqueKey
-                    );
-                    props.onDataBinSelection = this.handlers.onDataBinSelection;
+                    if (chartMeta.dataType === 'Custom_Data') {
```

Comment:
can we make "Custom_Data" into a constant somewhere?

Reply:
I think we can make use of the following:
```
if (  this.store.isUserDefinedCustomDataChart(
       chartMeta.uniqueKey
    )
```
In case of `ChartTypeEnum.BAR_CHART` and `isUserDefinedCustomDataChart` are both true, the custom data has a numerical type.

In other cases, we can make use of `ChartMetaDataTypeEnum.CUSTOM_DATA`